src/Build/Evaluation/Evaluator.cs

```diff
@@ -1360,6 +1360,7 @@ private void EvaluatePropertyElement(ProjectPropertyElement propertyElement)
                         !_data.GlobalPropertiesToTreatAsLocal.Contains(propertyElement.Name)
                     )
                 {
+                    _evaluationLoggingContext.LogComment(MessageImportance.Normal, "OM_GlobalProperty", propertyElement.Name);
```

Comment:
I can't decide whether I want this to be `Normal` or `Low` importance. I lean toward lower because I suspect it might be really chatty at higher levels, but that would reduce the usefulness to higher verbosity log levels.

Reply:
And maybe if there will ever be a strict mode (#3432), bump the importance when in strict mode.

One option is to explore: go with normal for now, and see what the response is before 16 ships.

Reply:
Works for me.